ansible/roles/operator-pipeline/files/operator-pipeline-config.yml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: operator-pipeline-config
```

Comment:
This does not seem to be the right name for this kind of config, see [here](https://github.com/tektoncd/pipeline/blob/main/docs/install.md#customizing-the-pipelines-controller-behavior). Also, I'm not sure this actually gets applied anywhere.